pyvcloud/vcd/vdc.py

```diff
@@ -415,6 +426,12 @@ def update_disk(self, name, size, new_name=None, description=None,
         else:
             disk = self.get_disk(name)
```

Comment:
I can see a bug in the implementation fo get_disk(name).

If there are two disks with the same name, we will return the first disk and never notify the user that there are more than 1 disk with the same name. We can address that in a new CL, but file a bug for the same.

Reply:
Hey ! Get disk does take disk_id as a param. 
get_disk(self, name, disk_id=None):
The use case you are pointing is already taken care of and this was demoed too by Paco (Same name disks with different disk_id)

Reply:
I am pointing to the case where there are two disks with the same name, and user supplies only the name of the vm (no vm_id is provided). In that case we just iterate over the disks in the vdc and the first disk that matches the name filter is returned, without ever prompting to the user that there are two disks with the same name.

As a CLI user you are not expected to remember how many disks are there in your inventory every time you run a command. Our CLI should point out that info to the user without updating a non desired disk.

What Paco demonstrated was the ability to distinguish between two disks with the same name. But what is missing is that we are not alerting/stopping the user from modifying a non-desired vm, which happens to be namesake of the vm we actually want to modify.